backend/src/main/java/com/eyetrade/backend/model/resource/annotation/AnnotationResource.java

```diff
@@ -0,0 +1,39 @@
+package com.eyetrade.backend.model.resource.annotation;
+
+import com.eyetrade.backend.constants.CommentType;
+import com.eyetrade.backend.model.entity.User;
+import com.eyetrade.backend.model.resource.user.MinimalUserResource;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.annotation.Resource;
+import java.util.UUID;
+
+/**
+ * Created by Emir Gökdemir
+ * on 8 Ara 2019
+ */
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@Resource
+public class AnnotationResource {
+    private UUID id;
+
+    private String content;
+
+    private MinimalUserResource user;
+
+    //article or event
+    private CommentType commentType;
+
+    private UUID articleEventId;
```

Comment:
From what I understand, here this field would have the corresponding article or event's ID that this annotation is created to. However, is it guaranteed that all articles and event's have different ID's?

Reply:
No it isn't. But it is guaranteed that all articles have different values and events too. We have also commentType section which is provide that we found correct instances